src/Nethermind/Nethermind.Merge.Plugin.Test/EngineModuleTests.HelperFunctions.cs

```diff
@@ -84,9 +84,12 @@ private ExecutionPayload CreateParentBlockRequestOnHead(IBlockTree blockTree)
             };
         }
 
-        private static ExecutionPayload CreateBlockRequest(ExecutionPayload parent, Address miner, IList<Withdrawal>? withdrawals = null, ulong? dataGasUsed = null, ulong? excessDataGas = null, Transaction[]? transactions = null)
+        private static ExecutionPayload CreateBlockRequest(ExecutionPayload parent, Address miner, IList<Withdrawal>? withdrawals = null, Transaction[]? transactions = null)
```

Comment:
Maybe just use specialized constructors on `ExcecutionPayload` and `ExcecutionPayloadV3` ?